package.json

```diff
@@ -1,53 +1,47 @@
 {
   "name": "opendistro-alerting",
-  "version": "1.11.0.2",
+  "version": "1.12.0.0",
   "description": "Kibana Alerting Plugin",
   "main": "index.js",
   "license": "Apache-2.0",
   "homepage": "https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin",
   "kibana": {
-    "version": "7.9.1",
-    "templateVersion": "6.3.3"
+    "version": "7.10.0"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin"
   },
   "scripts": {
     "kbn": "node ../../scripts/kbn",
-    "lint": "../../node_modules/.bin/eslint '**/*.js' -c .eslintrc",
+    "lint": "../../node_modules/.bin/eslint '**/*.js' -c .eslintrc --ignore-path .gitignore",
     "start": "plugin-helpers start",
-    "test:server": "plugin-helpers test:server",
-    "test:browser": "plugin-helpers test:browser",
     "test:jest": "../../node_modules/.bin/jest --config ./test/jest.config.js",
-    "build": "plugin-helpers build"
+    "build": "yarn plugin-helpers build",
+    "plugin-helpers": "node ../../scripts/plugin_helpers"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
   "devDependencies": {
-    "@elastic/eslint-config-kibana": "link:../../packages/eslint-config-kibana",
-    "@elastic/eslint-import-resolver-kibana": "link:../../packages/kbn-eslint-import-resolver-kibana",
-    "@kbn/expect": "link:../../packages/kbn-expect",
-    "@kbn/plugin-helpers": "link:../../packages/kbn-plugin-helpers",
     "husky": "^3.0.0",
     "lint-staged": "^9.2.0"
   },
   "dependencies": {
-    "brace": "^0.11.1",
```

Comment:
Why the change here?

Reply:
Kibana seems added a new limitation to have only one version for any dependencies in Kibana. (https://github.com/elastic/kibana/blob/e5d8d49164a8d993bdd314dd6c800f9186092148/packages/kbn-pm/src/utils/validate_yarn_lock.ts#L145)
Without the changes for the version ranges, an error with the above message will occur during `yarn kbn bootstrap`.
I'm not quite sure if the change is correct here.

Reply:
Looks like this is the only solution to deal with the new requirement for versions.